### ISSUE TITLE
Issue with Carthage

### ISSUE THREAD

**Issue by TofPlay** [xmartlabs/Bender#30]
Hi,
I just try install your component on a test project with Carthage
My `Cartfile`:
```
github "xmartlabs/Bender"
```
Carthage output:
```
$ carthage update --platform iOS
*** Cloning Bender
*** Checking out Bender at "0.1.0"
*** xcodebuild output can be found in /var/folders/rw/gbr9wvs91cz91mcpgdxv4q340000gn/T/carthage-xcodebuild.XMMgOU.log
*** Building scheme "Palladium" in Bender.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal armv7
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
(2 failures)
/Volumes/Perso/Temp/BenderTest/Carthage/Checkouts/Bender/Sources/Adapters/Tensorflow/protos/attr_value.pb.swift:10:8: error: no such module 'SwiftProtobuf'
A shell task (/usr/bin/xcrun xcodebuild -workspace /Volumes/Perso/Temp/BenderTest/Carthage/Checkouts/Bender/Bender.xcworkspace -scheme Palladium -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal armv7
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
(2 failures)
```
When I open the project I just see one iOS target 'Palladium' (why `Palladium` and not `MetalBender iOS` 😦)
If it's can help here a tutorial to create a cross platform framework compatible with macOS, iOS, tvOS and watchOS and with a clean support of Carthage: [Swift Cross Platform Framework](https://goo.gl/XxaJGo)
I hope this will help 😉

**Comment by AndreJFBico** [comment]
Hey if it helps you can try downloading swift protobuf from their repository directly and compiling the framework.
https://github.com/apple/swift-protobuf

There seemed to be some issues with carthage build.
Issue:
https://github.com/apple/swift-protobuf/issues/610

**Comment by 287258813** [comment]
you must have one more xcode,  just keep only one, and try again later

**Comment by bryant1410** [comment]
Fixed by #66
